src/drive/storage/index.ts

```diff
@@ -384,4 +385,15 @@ export class Storage {
   public getFolderMeta(uuid: string): Promise<FolderMeta> {
     return this.client.get<FolderMeta>(`folders/${uuid}/meta`, this.headers());
   }
+
+  /**
+   * Replaces a file with a new one.
+   *
+   * @param {string} uuid - UUID of the file.
+   * @param {ReplaceFile} payload
+   * @returns {Promise<DriveFileData>}
```

Comment:
Could you add a description of what it returns? Is it the new file? the replaced file? 🤔